app/statistics/server/lib/statistics.js

```diff
@@ -177,14 +178,71 @@ export const statistics = {
 		statistics.onHoldEnabled = settings.get('Livechat_allow_manual_on_hold');
 
 		// Number of Email Inboxes
-		statistics.emailInboxes = await EmailInbox.col.count();
+		statsPms.push(
+			EmailInbox.col.count().then((count) => {
+				statistics.emailInboxes = count;
+			}),
+		);
 
-		statistics.BusinessHours = {
-			// Number of Business Hours
-			total: await LivechatBusinessHours.col.count(),
-			// Business Hours strategy
-			strategy: settings.get('Livechat_enable_business_hours'),
-		};
+		statsPms.push(
+			LivechatBusinessHours.col.count().then((count) => {
+				statistics.BusinessHours = {
+					// Number of Business Hours
+					total: count,
+					// Business Hours strategy
+					strategy: settings.get('Livechat_enable_business_hours'),
+				};
+			}),
+		);
+
+		// Last-Chatted Agent Preferred (enabled/disabled)
+		statistics.lastChattedAgentPreferred = settings.get('Livechat_last_chatted_agent_routing');
+
+		// Assign new conversations to the contact manager (enabled/disabled)
+		statistics.assignNewConversationsToContactManager = settings.get('Omnichannel_contact_manager_routing');
+
+		// How to handle Visitor Abandonment setting
+		statistics.visitorAbandonment = settings.get('Livechat_abandoned_rooms_action');
+
+		// Amount of chats placed on hold
+		statsPms.push(
+			MessagesRaw.col
```

Comment:
I'm not really sure this would provide a real amount 👀 since a chat can be put on hold multiple times
a `distinct("rid")` would do the trick